flexmeasures/ui/templates/crud/users.html

```diff
@@ -1,26 +1,42 @@
-{% extends "base.html" %}
-
-{% set active_page = "users" %}
-
-{% block title %} User listing {% endblock %}
-
-{% block divs %}
+{% extends "base.html" %} {% set active_page = "users" %} {% block title %} User
+listing {% endblock %} {% block divs %}
 
 <div class="container-fluid">
   <div class="row">
     <div class="col-md-2"></div>
     <div class="col-md-8">
       <div class="card">
         <h3>All {% if not include_inactive %}active {% endif %}users</h3>
-        <form class="form-inline float-right" role="form" target="" method="get" id="user-list-options">
+        <form
+          class="form-inline float-right"
+          role="form"
+          target=""
+          method="get"
+          id="user-list-options"
+        >
           <div class="form-check form-check-inline">
             <label class="form-check-label">
-              <input name="include_inactive" {% if include_inactive %} checked="checked"{% endif %} type="checkbox"> Include inactive
+              <input
+                name="include_inactive"
+                {%
+                if
+                include_inactive
+                %}
+                checked="checked"
+                {%
+                endif
+                %}
+                type="checkbox"
+              />
+              Include inactive
             </label>
           </div>
         </form>
         <div class="table-responsive">
-          <table class="table table-striped table-responsive paginate nav-on-click" title="View/edit this user">
+          <table
+            class="table table-striped table-responsive"
+            title="View/edit this user"
```

Comment:
Right now, this is not true, the table is not clickable so I cannot view/edit the user. I'm no expert, but look for the `nav-on-click` class which was here before, and how that works.

Reply:
I finally figured this out, it wasn't working cause of the shift to handling this table with JS, but I've fixed it now and will ake a push shortly